vuepress/docs/next/docs/reference/deployment-structure.md

```diff
@@ -123,9 +126,9 @@ WAR files generated from a core build as dependencies. In a default deployment t
 * DockerHub: None (deployed to maven central)
 
 #### Customization
-For users familiar with versions prior to Entando 6 there will be cases where the _entando-core_ is customized. 
-In most cases these customizations will be delivered via WAR overlay in the instance project. 
-Using WAR overlay is a functional approach for users already  familiar with the process but it is highly 
+For users familiar with versions prior to Entando 6, there will be cases where the _entando-core_ is customized. 
+In most cases, these customizations will be delivered via a WAR overlay in the instance project. 
+Using WAR overlay is a functional approach for users already familiar with the process but it is highly 
```

Comment:
process, but (maybe not technically required but reads better with the pause)

Reply:
it is technically required here